libmattermost.c

```diff
@@ -2273,6 +2273,34 @@ mm_process_attachment(JsonObject *attachment)
 	return message;
 }
 
+static gint64
+mm_get_channel_approximate_view_time(MattermostAccount *ma, gchar *id)
+{
+	gchar *tmptime = NULL;
+
+	PurpleChat *chat = mm_purple_blist_find_chat(ma, id);
+	if (chat) {
+		tmptime = g_strdup(purple_blist_node_get_string(PURPLE_BLIST_NODE(chat),"channel_approximate_view_time"));
+	} else {
+		PurpleBuddy *buddy = purple_blist_find_buddy(ma->account,g_hash_table_lookup(ma->one_to_ones,id));

```

Comment:
```suggestion
		PurpleBuddy *buddy = purple_blist_find_buddy(ma->account, g_hash_table_lookup(ma->one_to_ones, id));
```
Also check which linter is used in this project and run it.